universal-login-sdk/test/e2e/authentication-relayer.ts

```diff
@@ -0,0 +1,109 @@
+import sinonChai from 'sinon-chai';
+import chai, {expect} from 'chai';
+import {Wallet, utils} from 'ethers';
+import {createFixtureLoader} from 'ethereum-waffle';
+import basicSDK from '../fixtures/basicSDK';
+import UniversalLoginSDK from '../../lib/sdk';
+import { CancelAuthorisationRequest, GetAuthorisationRequest, signCancelAuthorisationRequest, signGetAuthorisationRequest, createKeyPair} from '@universal-login/commons';
+import { RelayerUnderTest } from '@universal-login/relayer';
+
+const loadFixture = createFixtureLoader();
+
+chai.use(sinonChai);
+
+describe('E2E authorization - sdk <=> relayer', async () => {
+  let relayer: RelayerUnderTest;
+  let sdk: UniversalLoginSDK;
+  let contractAddress: string;
+  let privateKey: string;
+  let otherWallet: any;
+
+  const createGetAuthorisationRequest = (walletContractAddress: string, privateKey: string) => {
+    walletContractAddress.toLowerCase();
+    const getauthorisationRequest: GetAuthorisationRequest = {
+      walletContractAddress,
+      signature: ''
+    };
+    signGetAuthorisationRequest(getauthorisationRequest, privateKey);
+    return getauthorisationRequest;
+  };
+
+  beforeEach(async () => {
+    ({sdk, privateKey, contractAddress, relayer, otherWallet} = await loadFixture(basicSDK));
+  });
+
+  it('Send valid cancel request', async () => {
+    const {publicKey} = createKeyPair();
+    await sdk.relayerApi.connect(contractAddress, publicKey.toLowerCase());
```

Comment:
1) publicKey.toLowerCase() -> that should happen somewhere on relayer side
2) why not sdk.connect(...), those are e2e tests afterall